src/common/src/error.rs

```diff
@@ -70,6 +71,15 @@ impl Display for TrackingIssue {
     }
 }
 
+#[derive(Error, Debug, Macro)]
+#[error("Feature is not yet implemented: {feature}. {issue}")]
+#[thiserror_ext(macro(path = "crate::error"))]
```

Comment:
This is because we want the qualified name when calling from other crates to avoid importing the structure itself. So we generate the path `$crate::error::` before the type name.

If the structure is private, then `path` is not necessary.

BTW, is there any way to automatically derive this?

Reply:
`module_path!()`?

Reply:
I didn't find a way. 😕 Calling `module_path` in the proc macro body won't work as we will get the module path of the proc macro crate.